src/spn/algorithms/Inference.py

```diff
@@ -39,6 +39,21 @@ def prod_likelihood(node, children, data=None, dtype=np.float64):
     assert llchildren.dtype == dtype
     return np.prod(llchildren, axis=1).reshape(-1, 1)
 
+#added
```

Comment:
please use PEP8 standards for comment strings (also the line here is unnecessary)

Reply:
Yeah. I can make those changes